packages/demos/src/integrator-dom/integrator.ts

```diff
@@ -0,0 +1,24 @@
+import {createFeatureHub} from '@feature-hub/core';
+import {createFeatureAppLoader} from '@feature-hub/dom';
+import {loadAmdModule} from '@feature-hub/module-loader-amd';
+
+const {featureAppManager} = createFeatureHub('test:integrator', {
+  moduleLoader: loadAmdModule
+});
+
+createFeatureAppLoader(featureAppManager);
```

Comment:
maybe we should rename this method to `defineFeatureAppLoader` or `registerFeatureAppLoader`